src/components/HostSettingsButton.tsx

```diff
@@ -1,38 +1,40 @@
 'use client'
 
 import GearIcon from './icons/GearIcon'
+
 import { useRef } from 'react'
 
 type Props = {
 	roomId: string
 	roomUrl: string
 	defaultStoryPointValues: string[]
 	allowedPointsEmitter: (allowedStoryPoints: string[], localStorage: boolean) => void
+	allowedStoryPoints: string[]
 	setAllowedStoryPoints: (allowedStoryPoints: string[]) => void
 }
 
-// TODO: Need allowed story points saved in allowedStoryPoints state
-// to be passed to HostSettingsButton and have those values set
-// as checked in the form.
-
-// TODO: Need radio buttons for Hide Host Card: Yes/No
+// TODO Send output of Radio buttons for Hide Host Card to host
 
 export default function HostSettingsButton({
 	roomId,
 	roomUrl,
 	defaultStoryPointValues,
 	allowedPointsEmitter,
+	allowedStoryPoints,
 	setAllowedStoryPoints,
 }: Props) {
 	const dialogRef = useRef<HTMLDialogElement>(null)
 
 	function onSubmitForm(event: React.FormEvent<HTMLFormElement>) {
 		const formData = new FormData(event.currentTarget)
+		const hideHostCard = formData.getAll('hideHostCard')
 		const hostChosenPoints = formData.getAll('storyPoints')
-		allowedPointsEmitter(hostChosenPoints as string[], true)
-		setAllowedStoryPoints(hostChosenPoints as string[])
-
+		console.log('%c>>> hideHostCard', 'color: #f60', hideHostCard)
 		console.log('%c>>> hostChosenPoints', 'color: #5f0', hostChosenPoints)
+		if (hostChosenPoints.length > 0) {
+			allowedPointsEmitter(hostChosenPoints as string[], true)
+			setAllowedStoryPoints(hostChosenPoints as string[])
```

Comment:
nit: usually we type a value right at the source, rather than when using it. So in this case, you'd type directly when you create the `hostChosenPoints` :)

```js
		const hostChosenPoints = formData.getAll('storyPoints') as string[]
```

Reply:
Okay, that's actually good to know!! I've made the change. 

DONE!